salt/orch/update-etc-hosts.sls

```diff
@@ -0,0 +1,23 @@
+update_grains:
+  salt.function:
+    - tgt: 'roles:kube-(master|minion)'
+    - tgt_type: grain_pcre
+    - name: saltutil.refresh_grains
```

Comment:
Yes - it was added to the 2016.3.x branch (after it was forked), and the 2016.11.x branch in 2016.11.4 - if you look at the source in 2016.11.3 that function does not exist.